components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/ImportExportConstants.java

```diff
@@ -68,6 +68,10 @@ public final class ImportExportConstants {
     public static final String JSON_EXTENSION = ".json";
     public static final String YAML_EXTENSION = ".yaml";
 
+    public static final String SYNAPSE_POLICY_EXTENSION = ".j2";
```

Comment:
We already have these variables in APIConstants. 

https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java#L2880